src/arkode/arkode_erkstep_io.c

```diff
@@ -265,19 +477,55 @@ int ERKStepSetDefaults(void* arkode_mem)
     return(retval);
   }
 
+  /* Remove current SUNHeuristics object, and replace with "Default" */
```

Comment:
I didn't see similar updates to the ARKStep version of this function (or is this covered by the ARKODE defaults?)

Reply:
This is because in previous releases, the ARKODE "defaults" matched those in ARKStep, whereas ERKStep had a slightly different set of default parameters.  Thus in this PR, only ERKStep needs to clear out the controller/heuristics objects that were created/initialized by ARKODE, and then set these up directly.